dali/base/dadfs.cpp

```diff
@@ -8866,18 +8867,21 @@ class CFileScanner
             CFileMatch *match = &files.item(f);
             matchingFiles.append(*LINK(match));
             ++count;
+            if (checkFileCount && (count > ITERATE_FILTEREDFILES_LIMIT))
+                throw MakeStringException(-1, "More than %d files found, please use a filter to reduce the number of files returned.",
+                    ITERATE_FILTEREDFILES_LIMIT);
```

Comment:
I don't think any advantage in skipping the sorting (it will still look prettier).
But there needs to be a clear warning that there were too many matches, so this represents a sample only.